### PR TITLE
Eemep opath-r8.q

### PR DESCRIPTION
**Pull request by heikoklein**
Upgrade eemep to run on opath-r8.q

### CODE REVIEW

File: utils/SnapPy/METNO/PPI_R8B.py [hunk 1]
```diff
@@ -30,26 +30,26 @@
...
+    Implementation of a HPC machine for ppi-r8login-b1
     '''
 
 
     def __init__(self):
         '''
         Constructor
         '''
-        connection = SSHConnection(machine="xvis-m3b.met.no", port=22)
+        connection = SSHConnection(machine="ppi-r8login-b1", port=22)
```
Comment (magnusuMET) [inline utils/SnapPy/METNO/PPI_R8B.py:43]: ppi-r8login-b1.int.met.no

File: utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_r8b_direct_npp.job [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+## {runtag} {rundir} {year}-{month}-{day}T{hour} {runhour}
+
+#$ -N eemep_nuc
+#$ -l h_rt=01:30:00
+#$ -pe opa-fn 108
+#$ -l h_vmem=4G,h_rss=4G,mem_free=4G
+#$ -q opath-r8.q
+#$ -M beredskap-fou-kl@met.no
+#$ -P eemep
```
Comment (magnusuMET) [inline utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_r8b_direct_npp.job:10]: This project does not exist (yet)

File: utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_r8b_direct.job [hunk 3]
```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+## {runtag} {rundir} {year}-{month}-{day}T{hour} {runhour}
+
+#$ -N eemep
+#$ -l h_rt=01:30:00
+#$ -pe opa-fn 108
+#$ -l h_vmem=4G,h_rss=4G,mem_free=4G
+#$ -q opath-r8.q
+#$ -M beredskap-fou-kl@met.no
+#$ -P eemep
```
Comment (magnusuMET) [inline utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_r8b_direct.job:10]: This project does not exist (yet)

File: utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_r8b_direct_npp.job (hunk 2 above)
Comment (magnusuMET) [reply utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_r8b_direct_npp.job:10]: Account to `dsa`?

File: utils/SnapPy/METNO/PPI_R8B.py (hunk 1 above)
Comment (heikoklein) [reply utils/SnapPy/METNO/PPI_R8B.py:43]: thanks

File: utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_r8b_direct.job (hunk 3 above)
Comment (heikoklein) [reply utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_r8b_direct.job:10]: project eemep created

File: utils/SnapPy/METNO/PPI_R8B.py (hunk 1 above)
Comment (magnusuMET) [reply utils/SnapPy/METNO/PPI_R8B.py:43]: Seems to be missing still

**Review by magnusuMET — APPROVED** [review]

File: utils/SnapPy/METNO/PPI_R8B.py (hunk 1 above)
Comment (heikoklein) [reply utils/SnapPy/METNO/PPI_R8B.py]: now

**Pull request merged**